.github/workflows/ci.yaml

```diff
@@ -22,51 +48,47 @@ jobs:
           node-version: 18
 
       - name: Install Clean dependencies
-        working-directory: ./angular
         run: npm ci
 
       - name: Run headless tests
         id: run-tests
-        run: npm test -- --watch=false --browsers=ChromeHeadless 
-        working-directory: ./angular
-        continue-on-error: true
+        if: ${{ always() }}
+        run: npm run test -- --watch=false --browsers=ChromeHeadless 
 
       - name: Build Angular project
         id: build-project
         if: steps.run-tests.outcome == 'success'
```

Comment:
Try to not use these `steps.run-tests.outcome` conditions checks everywhere and check how it will behave. 

Basically the suggestion with `always` was about `Get pipeline execution time` and `Send email notification` steps. I.e. everything should run without condition by default in the same way as task suggest. I.e. workflow will fail on a first error (that is ok and ok for tests too), but `Get pipeline execution time` and `Send email notification` steps would run always.

Reply:
Thank you. Done